scripts/pre-commit.sh

```diff
@@ -10,7 +10,7 @@ if [ -n "$jsfiles" ]; then
     echo "Checking lint for:"
     for js in $jsfiles; do
         echo "$js"
-        e=$(node_modules/.bin/eslint --quiet $js)
+        e=$(node_modules/.bin/eslint --quiet $js --fix)
```

Comment:
🤔

Reply:
I was trying to commit and got a large list of dangling comma errors. So I thought it be nice if it can fix errors like that and let me fix the ones that it can't fix (like prop verification and stuff) and let eslint fix the small errors. or else i had to manually fix a huge list of dangling comma errors.